### ISSUE TITLE
Conflict with Cropsticks and Crook from Ex-Nihilo

### ISSUE THREAD

**Issue by shufunk** [AgriCraft/AgriCraft#1382]
## Quick description
The cropsticks recipe is conflicting with the crook from Ex-Nihilo

## Related issues and searches
None that I've seen. I may be the 1st?

## Mods and versions used
* Minecraft: 1.16.5
* AgriCraft: 3.0.4
* InfinityLib: 2.0.5
* Ex Nihilo Sequentia: 1.16-20210628182320
* JEI: 1.16.5-7.7.1.110

## Steps to replicate the issue
The normal 2x2 grid works but when using a crafting table you can put the sticks anywhere and it make the cropsticks. You can clearly see the recipe for crook in JEI but does not work due to confliction

## Expectations and explanations
I need to make a crook to start in a sky block type of world. If any of you played SkyFactory or any other modpack like that you know what i am talking about

## Your environment
* How are you playing?
  * Operating System: Windows 10
  * Launcher: Curseforge Beta
  * Modpack: ShuFactory 3 (in the works not officialy uploaded
* What kind of server:
  * [X ] Singleplayer
  * [ ] Multiplayer
* Game Modes:
  * [X ] Survival
  * [ ] Creative
* Save File:
  * [X ] Brand new save
  * [ ] Only an existing save

## Links to crash logs, stack traces, etc
N/A

## Screenshots (or videos)

https://youtu.be/WakExgjn-04

**Comment by InfinityRaider** [comment]
This is a recipe conflict between two mods, this is something you need to fix with a datapack or CT

**Comment by darthvader45** [comment]
Or Polymorph (based off of NoMoreRecipeConflict by stimmedcow): https://www.curseforge.com/minecraft/mc-mods/polymorph

**Comment by shufunk** [comment]
> 
> 
> This is a recipe conflict between two mods, this is something you need to fix with a datapack or CT

Ok but this is still a problem with the mod by itself! The item needs to be one recipe and not be able to do it in any other shapes. 

![2021-07-12_17 23 00](https://user-images.githubusercontent.com/15242366/125358027-678d8880-e336-11eb-82e9-f010d7fe752d.png)
Above is how it should be and none of the below should be possible. 

![2021-07-12_17 23 05](https://user-images.githubusercontent.com/15242366/125358028-68261f00-e336-11eb-93b0-44adfc28d8b6.png)
![2021-07-12_17 23 08](https://user-images.githubusercontent.com/15242366/125358030-68261f00-e336-11eb-9cc3-4517a48ff4b7.png)
![2021-07-12_17 23 10](https://user-images.githubusercontent.com/15242366/125358031-68beb580-e336-11eb-9daa-b1bfe3ed6f73.png)
![2021-07-12_17 23 13](https://user-images.githubusercontent.com/15242366/125358032-68beb580-e336-11eb-988b-82d1aafc928e.png)
![2021-07-12_17 23 18](https://user-images.githubusercontent.com/15242366/125358034-69574c00-e336-11eb-8c1c-5465eca6bdea.png)
![2021-07-12_17 23 20](https://user-images.githubusercontent.com/15242366/125358035-69574c00-e336-11eb-8415-3fc0d67662d6.png)

**Comment by InfinityRaider** [comment]
This is not a problem with the mod itself, I am using a shapeless recipe, which if you don't like, can perfectly be modified with a datapack.

**Comment by shufunk** [comment]
That's cool i will figure it out. Just with my last modpack with agricraft and exnihlo for MC 1.12.2 I never had to do any major changes. As a person (me) who don't understand the code behind mods maybe this datapack editing was built in or maybe your older version was not shapeless? Either way I was only wanting to point this out and thanks again for telling me how to work around this not a bid deal I love learning new things.

**Comment by InfinityRaider** [comment]
I think vanilla code changes might have provoked this, as crop sticks have always been shapeless.
Imo shaped recipes should have priority over shapeless recipes, but alas, that was not the case.

That being said, it is unfeasible for me to go and modify recipe conflicts with every mod out there.
I would recommend CraftTweaker to modify the crop sticks or crook recipe as desired.

**Comment by shufunk** [comment]
That had to be it. Vanilla is always changing and I know mod makers gotta adapt and learn of new things that may  happen without knowing till it happens. I will be using CraftTweaker after looking into it. I have a lot of down time at work so will work on this. As for a datapack I did learn of a program that converts a datapack into a mod so it helps so people don't have to load the data pack manually with every single map they create. If curse allows a special mod that's unique to a pack i may go that route. Both ways will work just gotta choose which way i wanna do it for my modpack. Thanks for being cool with me and explaining this. At least we learned something new from this.
